### ISSUE TITLE
[QUESTION]: Not found function【get_vector_ids 】in v2.0.0 ，which is implemented in v1.x

### ISSUE THREAD

**Issue by lllfx** [milvus-io/pymilvus#888]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What is your question?

Not found function【get_vector_ids 】in v2.0.0 ，which is implemented in v1.x

`@error_handler([])
    def get_vector_ids(self, collection_name, segment_name, timeout=30):
        request = grpc_types.GetVectorIDsParam(collection_name=collection_name,
                                               segment_name=segment_name)

        rf = self._stub.GetVectorIDs.future(request, wait_for_ready=True, timeout=timeout)
        response = rf.result()

        if response.status.error_code == 0:
            return Status(), list(response.vector_id_array)
        return Status(response.status.error_code, response.status.reason), []`

It's implemented in v1.x, but not in v2.0

**Comment by XuanYang-cn** [comment]
@lllfx Yes, Milvus 2.0 doesn't support getting all IDs of a segment, maybe you could file an issue in Milvus to request such a feature.
